performance/SwiftUIPerformanceQuickstart/PerformanceExample/Shared/Process.swift

```diff
@@ -23,27 +23,43 @@ import Vision
 class Process: ObservableObject {
   @Published var status: ProcessStatus = .idle
   @Published var image: UIImage?
+  @Published var saliencyMap: UIImage?
```

Comment:
For easier portability later on, it may be easier to make this a `CIImage` and then have platform-specific `lazy var`s that return UIImage or NSImage as necessary.

Reply:
Should I add it to this PR? I imagine we might also want to do the same for `image: UIImage?`, so perhaps I can do this as a follow-up PR?

Reply:
Will add as follow-up PR.